src/puppet-padplus.ts

```diff
@@ -1532,7 +1548,8 @@ export class PuppetPadplus extends Puppet {
       throw new Error(`no manager.`)
     }
     await this.manager.setRoomTopic(roomId, topic as string)
-    await this.roomPayloadDirty(roomId)
+    await this.dirtyPayload(PayloadType.Room, roomId)
```

Comment:
Why need double `dirtyPayload()`?

> `dirty` event will call `puppet.dirtyPayload()`.

Reply:
I thought we need to clear cache in `wechaty-puppet-padplus` layer, I misunderstood the code.